typescript/packages/jumble/src/contexts/CharmManagerContext.tsx

```diff
@@ -29,16 +29,7 @@ export const CharmsManagerProvider: React.FC<{ children: React.ReactNode }> = ({
   }
 
   const charmManager = useMemo(() => {
-    const storageType = (import.meta as any).env.VITE_STORAGE_TYPE ?? "remote";
-    const storage =
-      storageType === "remote"
-        ? createStorage({
-            type: "remote",
-            replica: effectiveReplica,
-            url: new URL(location.href),
-          })
-        : createStorage({ type: storageType as "memory" | "local" });
-    return new CharmManager(storage);
+    return new CharmManager(effectiveReplica);
```

Comment:
I was thinking perhaps we could have replica names __memory__ or __local__ to support using local/memory storage?

Reply:
Ups, actually just removed LocalStorage, since it was behind in other ways. We could bring it back at anytime though. 

Just this morning I found it useful to swap `remote` for `memory` to debug something.

Either way, since it looks like spaces are (at least overlapping with) a user-facing concept of organizing stuff, this is probably more a property of a space than a dedicated space?